src/initiatives.njk

```diff
@@ -1,6 +1,9 @@
 ---
 title: Initiatives
-permalink: /initiatives/
+permalink: /initiatives/{% if pagination.pageNumber > 0 %}page/{{ pagination.pageNumber + 1 }}/{% endif %}
```

Comment:
Without having the pagination bar on the page for users to switch between pages, adding pagination to `permalink` doesn't make sense.

Reply:
Okay, this line change should happen in a separate PR where pagination is added to the initiatives page.

Reply:
This will be addressed in this [issue](https://github.com/inclusive-design/wecount.inclusivedesign.ca/issues/383)